### ISSUE TITLE
Daily meeting ( 13/12/2016)

### ISSUE THREAD

**Issue by Lai0723** [Lai0723/Gallery-Shopping-Website#2]
Yesterday i just finish with the database for the member login and product(included category), you all can go have a look and tell me if got any problem if no you all can proceed with your part first. So today i will continue to develop the database for the cart.

**Comment by boonsenglim** [comment]
Wow, well progress Wei Chun, i just finish design the registration form and now working on the validation part.

**Comment by pqyyieeuyi** [comment]
May I ask, why you need validate on the registration part?

**Comment by boonsenglim** [comment]
I do that because to force user to user password that not too simple such as 123456 since this will effect the security of the website.

**Comment by kcchan3012** [comment]
I agree with boon seng , security for a website is very important. I just create a menu page and product detail page and anyone got any idea what colour should we use on the website?

**Comment by Lai0723** [comment]
Erm, i think black and white is the best because simple and clean.

**Comment by pqyyieeuyi** [comment]
I not agree with wei chun , balck and white is too simple and cannot attract user attention. I think need to use more colour.

**Comment by boonsenglim** [comment]
How about we use the orange colour on the category tab that the user choose to view different type of gallery?

**Comment by kcchan3012** [comment]
That seem quite interesting , we can give it a try.

**Comment by pqyyieeuyi** [comment]
I just finished design what the login form should have, only need username and password right nothing else?

**Comment by kcchan3012** [comment]
You can include the hyperlink below the function in order to let them register a new account to give them convenient.

**Comment by pqyyieeuyi** [comment]
ok, i will add in now.

**Comment by Lai0723** [comment]
So i think that all for today meeting, everyone can dismiss now.
